src/main/java/teammates/common/util/Const.java

```diff
@@ -253,14 +248,12 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String COURSE_REMIND = "Email an invitation to all students yet to join requesting them to join the course"
                 + " using their Google Accounts. Note: Students can use TEAMMATES without ‘joining’, but a joined student can access"
                 + " extra features e.g. set up a user profile";
-        public static final String COURSE_DELETE_ALL_STUDENTS = "Delete all students in this course";
```

Comment:
doesn't seem to be used at any point of time

added in:
~~~
commit b46c50c7c818ccbe09e547e9fc0b827ef4f102cc
Author: aldrianobaja.m@gmail.com <aldrianobaja.m@gmail.com>
Date:   Tue Jun 12 19:47:16 2012 +0800

    Modularize the servlets
    Added Evaluation Results page
    Initialize test on Evaluation Results page
    Fixed test on Evaluation Add page, sensitive to date, used regex.
    Extracted statusMessage.jsp
    Added errorPage.jsp
    Moved some deprecated testing function back to SharedLib
~~~